ui/admin/app/routes/scopes/scope/targets.js

```diff
@@ -69,6 +69,33 @@ export default class ScopesScopeTargetsRoute extends Route {
     this.refresh();
   }
 
+  /**
+   * save target with filters based on egress/ingress toggle
+   * @param {object} target
+   * @param {boolean} egressEnabled
+   * @return {Promise}
+   */
+  @action
+  @loading
+  async saveWithToggles(target, egressEnabled = true) {
```

Comment:
Is there an advantage to having a separate save function that calls into the original save instead of just using the same save?

Beyond that, how come this logic doesn't live in the the save function in the form rather than here in the route?